hr_employee_ppe/models/hr_employee_ppe.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 Escodoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class HrEmployeePPE(models.Model):
+    """Adds PPE information and allocation."""
+
+    _name = 'hr.employee.ppe'
+    _description = 'Personal Protective Equipments'
+    _inherit = ['mail.thread', 'mail.activity.mixin']
+
+    name = fields.Char(
+        required=True,
+    )
+    ppe_id = fields.Many2one(
+        string='Equipment',
+        required=True,
+        comodel_name='hr.employee.ppe.equipment',
+    )
+    employee_id = fields.Many2one(
+        comodel_name='hr.employee',
+        string='Employee',
+        required=True
+    )
+    start_date = fields.Date(
+        string='Start Date',
+        default=fields.Date.today()
+    )
+    end_date = fields.Date(
+        string='End Date'
+    )
+    description = fields.Text(
+        string='Description'
+    )
+    partner_id = fields.Many2one(
+        comodel_name='res.partner',
+        string='Issued By',
+        help='Certification Authority'
+    )
+    location = fields.Char(
+        string='Location',
+        help='Location'
```

Comment:
Please, give some more information on help: eg. is it where to use them? Is it where to get them?

Reply:
Actually I think I will change this field to something like "Indications" to point out the intended use for the specific employee.